bedrock/firefox/templates/firefox/features/safebrowser.html

```diff
@@ -0,0 +1,171 @@
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans url='https://support.mozilla.org/kb/use-master-password-protect-stored-logins' %}
+        Create secure passwords for every account and Password Manager will remember them all. We recommend creating a <a href="{{ url }}">Master Password</a> to safeguard all of your passwords in case your computer is lost or stolen.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Firefox Lockwise') }}</h2>
+      <p>
+        {% trans %}
+        You're carrying a fairly powerful tiny computer in your pocket. You might know it as your phone. It's got its
+        own internet browser, apps that connect to the web, passwords, and logins. You use it in coffee shops and other
+        public places that make it easy for hackers to steal your information.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans url='https://lockwise.firefox.com/' %}
+        On <a href="{{ url }} ">Firefox Lockwise</a>, you can take your passwords with you and not worry about their security. They’re all in one place and protected on desktop, phone or tablet.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Tracking Protection') }}</h2>
+      <p>
+        {% trans %}
+        Being followed doesn’t make anyone feel safe, even if it’s an annoying ad following you on the web. Blocking third-party trackers can help by not allowing the annoying “cookies” tracking that enables ads.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans url='https://blog.mozilla.org/firefox/control-trackers-with-firefox/' %}
+        That’s why we give you the <a href="{{ url }}">option to block third-party trackers</a> and content blocking on private and regular browsing mode.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Private Browsing') }}</h2>
+      <p>
+        {% trans %}
+        Speaking of private browsing, it’s a powerful privacy tool that’s easy to use. On Firefox, private browsing means we don't save information about where you’ve been, clicked or looked. Private browsing leaves smaller digital footprints, giving away less about what you do and where you go online.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Containers') }}</h2>
+      <p>
+        {% trans %}
+        Cleaning up your online space isn’t just good feng shui, it can lessen the worry of being tracked. Separate identities for your work websites or your personal interests keep trackers confined to one area. Ads that have nothing to do with work won’t pop-up in the middle of a presentation. For this purpose, we created Containers.
```

Comment:
I don't know the l10n plan for this page, I suspect it might never be translated, but we should add an l10n comment about about "feng shui," maybe "Feng shui is a traditional Chinese practice meant to bring harmony to one's environment. This is an anglicized form of a Chinese word and may be difficult to translate. Alternative: 'good housekeeping'."

Reply:
thank you for the suggestion, this is super helpful.